### DISCUSSION TITLE
AspNet Boilerplate allows to seed tenant  ,How to, seed tenant on abpframework?

### DISCUSSION THREAD

**Discussion by mrs01dev** [abpframework/abp#6857] [Q&A]
AspNet Boilerplate seed tenant through code "DefaultTenantBuilder" on "EntityFrameworkCore" project.
How to, seed tenant on abpframework?

**Comment by maliming** [comment]
Please check the document first. 
https://docs.abp.io/en/abp/latest/Data-Seeding
